eng/common/cross/toolchain.cmake

```diff
@@ -71,8 +71,19 @@ endif()
 
 if(TARGET_ARCH_NAME MATCHES "^(arm|armel)$")
   add_compile_options(-mthumb)
-  add_compile_options(-mfpu=vfpv3)
-  if(TARGET_ARCH_NAME STREQUAL "armel")
+  set(FPUFLAG vfpv3)
+  if (FEATURE_ARM_FPU_SPEC)
```

Comment:
I think you make this ever simpler and allow FPUFLAG to be specified on the command line directly. Then the whole change needed here is just just set check for empty `FPUFLAG` around `set(FPUFLAG vfpv3)`

Reply:
I can do that. any preference on names?